src/plugins/jartic/data_parser.py

```
@@ -0,0 +1,539 @@
+import zipfile
+from datetime import datetime, timedelta, timezone
+from decimal import Decimal
+from io import StringIO
+from pathlib import Path
+from typing import Any, AsyncIterator, Dict, List, Optional
+
+from ...domain.models import (
+    Coordinates,
+    Location,
+    Measurement,
+    MeasurementUnit,
```

Ruff (0.12.2)

`...domain.models.MeasurementUnit` imported but unused

Remove unused import: `...domain.models.MeasurementUnit`

(F401)